ROADMAP.md

```
@@ -0,0 +1,39 @@
+# Sentrix Roadmap
+
+## Status Legend
+- ✅ Done | 🔄 In Progress | 📋 Planned | 🔮 Future
+
+## Q2 2026
+
+### Core Protocol
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)